scripts/extensions.sh

```diff
@@ -6,23 +6,36 @@
 
 set -e
 
-STATIC_EXTENSIONS=(sqlsrv swoole)
+function install_extensions {
+    local PHP=$1
+    local -a EXTENSIONS=("${@:2}")
+
+    case "$PHP" in
+        8.1)
+            echo "Cannot install extension for the current PHP version."
+            echo "Please use \".laminas-ci/pre-run.sh\" to setup specific extensions for PHP $PHP"
+            echo "Additional details can be found on https://stackoverflow.com/q/8141407"
+            echo "The following extensions were not installed: ${EXTENSIONS[@]}"
```

Comment:
Add `# shellcheck disable=SC2145` above this line - this is an approrpriate place to expand the array within quotes.

Reply:
I've used the suggestion from shellcheck (`*` instead of `@`).

Reply:
Works. :)